contracts/BitGuildWhitelist.sol

```diff
@@ -0,0 +1,86 @@
+pragma solidity ^0.4.22;
+
+import "./BitGuildAccessAdmin.sol";
+
+
+/**
+ * @title BitGuildWhitelist
+ * @dev A small smart contract to provide whitelist functionality and storage
+ */
+contract BitGuildWhitelist is BitGuildAccessAdmin {
+    mapping (address => bool) public isWhitelisted;
+
```

Comment:
Nit: may be worth adding total count just for the tracking purposes btw - based on my experience writing our crowdsale whitelist. To know how many are whitelisted.

Reply:
Good suggestion. Added.